src/MSBuild/TerminalLogger/TerminalLogger.cs

```diff
@@ -939,6 +940,9 @@ private void WarningRaised(object sender, BuildWarningEventArgs e)
         _immediateMessageKeywords.Any(imk => message.IndexOf(imk, StringComparison.OrdinalIgnoreCase) >= 0);
 #endif
 
+
+    private bool IsCopyTaskRetryCode(string code) => code == "MSB3026";
```

Comment:
I suggest having a more generic name here, maybe something like

```suggestion
    private bool IsImmediateWarning(string code) =>
        // Copy task retries can often be resolved by closing an app, so surface them immediately.
        code == "MSB3026";
```

Reply:
done